src/mjolnir/graphtilebuilder.cc

```diff
@@ -11,6 +11,13 @@
 #include <set>
 #include <stdexcept>
 
+#include "baldr/directededge.h"
+#include "baldr/graphconstants.h"
+#include "baldr/graphid.h"
+#include "baldr/graphtileheader.h"
+#include "mjolnir/edgeinfobuilder.h"
+#include "mjolnir/landmark_builder.h"
```

Comment:
i am certain some of these are redundant. in fact i'd be the last one is the only one you need